### ISSUE TITLE
Move slot component button on worker inspector to left side to prevent accidental component deletion

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#2657]
### Is your feature request related to a problem? Please describe.

Where the button is now could result in accidental component deletion
If you have shaky hands in VR you could accidentally delete a component
![image](https://github.com/user-attachments/assets/8dba7e32-94a8-46c0-a4c3-7e55f0e79e90)

### Describe the solution you'd like

I want the button to be here as shown below
This both looks better and prevents accidental component deletion
![image](https://github.com/user-attachments/assets/e43bc1fc-41f3-4624-824a-fc375dd7ffef)

### Describe alternatives you've considered

Not having shaky hands or accidentally clicking the delete button

### Requesters

@epicEaston197

**Comment by BlueCyro** [comment]
This looks easy enough to do real quick

**Comment by Frooxius** [comment]
I don't think this is necessarily the best way to solve this problem. We generally put all the buttons on the right, so breaking the convention specifically to solve that problem might not be the best idea, because it doesn't address the underlying problem.

Instead, what might be better approach in general is to add confirmation to deletions, since this would help accidental deletions in other cases as well.

**Comment by epicEaston197** [comment]
> I don't think this is necessarily the best way to solve this problem.

I think it generally looks better so I'll just wait until the inspector rework to fix this 
A confirmation dialog thing would be nice though

**Comment by Frooxius** [comment]
Merged in the confirmation for the press which should help prevent accidental deletes in 2025.1.30.1401! Thanks!
